RemoteWellnessSupportApp/OnBoarding/OnboardingScreenView.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  OnboardingScreenView.swift
+//  RemoteWellnessSupportApp
+//
+//  Created by 岩本雄貴 on 2024/01/31.
+//
+
+import SwiftUI
+
+struct OnboardingScreenView: View {
+    @AppStorage("hasCompletedOnboarding") var hasCompletedOnboarding: Bool = false
+    @State private var selection = OnBoardingTab.introduction.rawValue
+
+    var body: some View {
+        VStack {
+            TabView(selection: $selection) {
+                IntroductionView()
+                    .tag(OnBoardingTab.introduction.rawValue)
+                ReminderExplanationView()
+                    .tag(OnBoardingTab.reminder.rawValue)
+                HealthReviewDescriptionView()
+                    .tag(OnBoardingTab.healthReview.rawValue)
+                ReviewDescriptionView()
+                    .tag(OnBoardingTab.review.rawValue)
+                WatchFeatureExplanationView()
+                    .tag(OnBoardingTab.watchFeature.rawValue)
+            }
+            .tabViewStyle(PageTabViewStyle())
+            .indexViewStyle(PageIndexViewStyle(backgroundDisplayMode: .always))
+
+            if selection == OnBoardingTab.watchFeature.rawValue {
```

Comment:
selectionをEnumにすると、比較する時もEnumが使えます。

```suggestion
            if selection ==.watchFeature {
```

Reply:
> selectionをEnumにすると、比較する時もEnumが使えます。

なるほど、知らなかったです。
本件もこちらで修正しました
https://github.com/y-iwamoto/RemoteWellnessSupportApp/pull/3/commits/09eabd5de4e18171a363e721c62222aa8dd55009